packages/docs-data/docsUtils.js

```diff
@@ -6,7 +6,7 @@ const marked = require("marked");
 const Highlights = require("highlights");
 
 const DEFAULT_SCOPE = "source.tsx";
-const HIGHLIGHTS_LANGUAGES = ["better-handlebars", "language-typescript", "language-less"];
+const HIGHLIGHTS_LANGUAGES = ["better-handlebars", "language-less", "tree-sitter-typescript"];
```

Comment:
where do we actually use handlebars in code examples?

Reply:
every html snippet, for `{{:modifier}}` support